ops/model.py

```diff
@@ -622,3 +667,21 @@ def storage_add(self, name, count=1):
         if not isinstance(count, int) or isinstance(count, bool):
             raise TypeError(f'storage count must be integer, got: {count} ({type(count)})')
         self._run('storage-add', f'{name}={count}')
+
+    def function_get(self):
+        command_name = 'function-get'
+        if not shutil.which(command_name):
+            command_name = 'action-get'
+        return self._run(command_name, return_output=True, use_json=True)
+
+    def function_set(self, **kwargs):
+        command_name = 'function-set'
```

Comment:
Is it worth caching the command name rather than doing a $PATH lookup on every call to function_set() ? It doesn't change for the lifetime of a process, and the LazyMapping means that every call to "result.foo = X" is going to trigger this.